test/mongodb.test.js

```diff
@@ -126,11 +171,11 @@ describe('mongodb', function () {
       should.not.exist(err);
       should.not.exist(person.id);
       person._id.should.be.equal(3);
-      
+
```

Comment:
If you rebase your code on top of master all of these linting issues are already fixed.